lib/index.js

```diff
@@ -1672,7 +1672,9 @@ Stream.prototype.write = function (x) {
 /**
  * Forks a stream, allowing you to add additional consumers with shared
  * back-pressure. A stream forked to multiple consumers will only pull values
- * from its source as fast as the slowest consumer can handle them.
+ * from its source as fast as the slowest consumer can handle them. For an
+ * example of the difference between `fork` and `observe`, see [this
+ * post](https://github.com/caolan/highland/issues/561#issuecomment-257069894).
```

Comment:
Is this needed if you include an example below?

Reply:
Good point. I've incorporated more of the example from that comment directly into the docs and then removed the link to the comment.